index.js

```diff
@@ -873,6 +879,38 @@ class KeyringController extends EventEmitter {
       );
     }
   }
+
+  async _newKeyring(type, data) {
```

Comment:
Nit: Could you add a JSDoc comment for this function?

Reply:
Also for `keyringBuilderFactory`

Reply:
Done